pom.xml

```diff
@@ -74,6 +75,23 @@
             <name>Confluent</name>
             <url>${confluent.maven.repo}</url>
         </repository>
+        <repository>
+            <id>maven-central</id>
+            <name>Maven Central</name>
+            <layout>default</layout>
+            <url>http://repo1.maven.org/maven2</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
```

Comment:
The Confluent Maven repo should proxy anything in Maven Central, so this should be unnecessary.

Reply:
I got an error if removed Maven Central repo.

```
[ERROR] Failed to execute goal on project kafka-connect-jdbc: Could not resolve dependencies for project io.confluent:kafka-connect-jdbc:jar:5.1.0-SNAPSHOT: The following artifacts could not be resolved: javax.ws.rs:javax.ws.rs-api:jar:2.1, org.xerial:sqlite-jdbc:jar:3.8.11.2, org.postgresql:postgresql:jar:9.4-1206-jdbc41, net.sourceforge.jtds:jtds:jar:1.3.1, org.easymock:easymock:jar:3.6, org.powermock:powermock-module-junit4:jar:2.0.0-beta.5, org.powermock:powermock-api-easymock:jar:2.0.0-beta.5, org.apache.derby:derby:jar:10.11.1.1, org.mockito:mockito-all:jar:1.10.19, com.github.dblock.waffle:waffle-jna:jar:1.7.5, net.java.dev.jna:jna:jar:4.2.1, net.java.dev.jna:jna-platform:jar:4.2.1, org.slf4j:jcl-over-slf4j:jar:1.7.12, com.fasterxml.jackson.core:jackson-databind:jar:2.5.3, org.osgi:org.osgi.enterprise:jar:4.2.0: Could not find artifact javax.ws.rs:javax.ws.rs-api:jar:2.1 in confluent (http://packages.confluent.io/maven/) -> [Help 1]
```